app.rb

```diff
@@ -1,8 +1,38 @@
-require 'sinatra/base'
+require "sinatra/base"
+require "sinatra/reloader"
+require "./lib/peep"
 
 class Chitter < Sinatra::Base
-  get '/test' do
-    'Test page'
+  configure :development do
+    register Sinatra::Reloader
+  end
+
+  get "/test" do
+    "Test page"
+  end
+
+  get "/" do
+    @peeps = Peep.all
+    erb(:index)
+  end
+
+  get "/new" do
+    erb(:new)
+  end
+
+  post "/" do
```

Comment:
I'm not sure if having two routes with the '/' path is confusing? Maybe this could be called '/post-peep' or something that then redirects to '/'